.github/workflows/run_tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pyats[full]
+        pip install pyats[full]==20.8
```

Comment:
I think they fixed this already, will likely have to revert in order to have PR accepted. 

Looks good overall though, once addressed, go ahead and create PR to Cisco